README.md

```diff
@@ -1,194 +1,29 @@
-# Red Hat AppStudio E2E Tests and Testing Framework
+# RHTAP E2E framework
```

Comment:
why "E2E Tests" was removed from the title?

Reply:
We are not building a E2E framework which contains E2e tests?

Reply:
E2E framework is not a mechanism which drive tests?

Reply:
Yes, but now the title makes it look like the e2e-tests repo **only** contains the framework and not the tests along with it.

Reply:
Ok, lets add it. Anyway dont make to much sense to me to be so explicitly